package.json

```diff
@@ -30,7 +32,7 @@
     "build:test": "webpack --config webpack.test.config.js",
     "prestart": "npm run build:dev",
     "start": "grunt start-server-open",
-    "prepublish": "grunt prep-release",
+    "prepublish": "grunt prep-release && webdriver-manager update",
```

Comment:
I guess it takes less than a second if the file is already downloaded, so I'll put it right before we run the tests